Makefile

```diff
@@ -50,13 +50,17 @@ else
   VALIDATE_FLAG =
 endif
 
+BUILD_PPRZLINK++ ?= OFF
+
 all: libpprzlink
 
 # Build and install C++ if lib versoin is 2.0 (Should test if version is more than 2.0...)
+ifeq ($(BUILD_PPRZLINK), ON)
 ifeq ($(PPRZLINK_LIB_VERSION), 2.0)
 libpprzlink: libpprzlink++
 libpprzlink-install: libpprzlink++-install
 endif
+endif
```

Comment:
Why not completely removing this, so C++ version is only compiled if explicitly called with `make libpprzlink++` ?
Same as suggested but with less lines in makefile.

Reply:
yep, I do it.